### PR TITLE
Prepare 0.9.0 release

### PR DESCRIPTION
**Pull request by lopuhin**
I'll make 0.9.0 release once this PR is merged into master. Added a short overview of release procedure.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.9.2 (2019-07-05)
```
Comment (kmike) [inline CHANGES.rst]: shouldn't it be 0.9.0?

File: docs/source/contribute.rst [hunk 2]
```diff
@@ -17,5 +17,16 @@ from source checkout.
 
 We like high test coverage and mypy_ type annotations.
 
+Making releases
+---------------
```
Comment (kmike) [inline docs/source/contribute.rst]: What do you think about adding a note that contributors shouldn't do this themselves, that this is a memo for the maintainers? It is inconvenient to receive pull requests which bump package version, in addition to implementing a feature.

Comment (lopuhin) [reply docs/source/contribute.rst]: Good point. What do you think of the following right after the header:

```
Note: releases are made from master by eli5 maintainers.
When contributing a pull request, please do not update release notes
or package version.
```

Comment (kmike) [reply docs/source/contribute.rst]: :+1:

**Comment by kmike** [comment]
Thanks for taking care of the release @lopuhin!

**Comment by lopuhin** [comment]
Thank you for review @kmike ! Released https://pypi.org/project/eli5/ and tagged.
